### PR TITLE
Newsweek clone feature branch

### PR DESCRIPTION
**Pull request by mosesmuchemi**
Creating a Pull request for peer to peer code review, stickler and TSE review

### CODE REVIEW

**Comment by shloch** [comment]
### Pull request under review by SHEY LOUIS

**Comment by shloch** [comment]
Great Job @mosesmuchemi  👏 👏 
Design looks good, adapts very well to the breakpoints and mimics the original pretty well at many levels (but not all). I require you to bring some changes therefore :

**CHANGES REQUIRED:**
1-  There're validation errors (please validate your html syntax and semantics here -> `https://validator.w3.org/nu/#textarea` )

2 - Try to mimic Better this Header design

original image at break points around 900px (date disappears, and 'NEWSWEEK' moves to right)
Screen Shot 2019-11-13 at 6 33 56 PM

Your own image
Screen Shot 2019-11-13 at 6 34 04 PM

I know you you did it at a lower breakpoint (around 700px)

3 - "MORE STORIES" SECTION

original website  (notice the background color):
Screen Shot 2019-11-13 at 6 46 12 PM

Your own display
Screen Shot 2019-11-13 at 6 46 29 PM

4 - "IN THE MAGAZINE" SECTION

original website: (notice the RED-color titles under the images)
Screen Shot 2019-11-13 at 6 48 27 PM


Your own version
Screen Shot 2019-11-13 at 6 48 36 PM

5 - 'FEATURED SLIDESHOWS' section

original version (notice the spaces/padding/gutters around the block)
Screen Shot 2019-11-13 at 6 53 27 PM

your own version (yours doesn't have the gutter at an important breakpoint around 1100px )
Screen Shot 2019-11-13 at 6 53 35 PM

Monitor and correct these spaces on other sections as well, you will notice more

6 - FOOTER SECTION

original version  (notice the links below "NEWSWEEK" take up about 50% of the page, horizontally)
Screen Shot 2019-11-13 at 6 58 43 PM

your own version (yours is about 25-30%)
Screen Shot 2019-11-13 at 6 59 01 PM

7 - Error display on small screens

original version :  (notice at small screens, TOP STORIES passes to the top )
Screen Shot 2019-11-13 at 7 06 24 PM

Your own version :

You have but "FEATURED STORIES"
Screen Shot 2019-11-13 at 7 06 34 PM

And your "TOP STORIES" is pushed down and doesn't display properly 
Screen Shot 2019-11-13 at 7 10 36 PM

Some  sections are even cut (still on the "TOP STORIES" section), please take a look at this very well 
Screen Shot 2019-11-13 at 7 11 28 PM

It long project, it's impressive what you have brought up so far.

I am also impressed at your style of reproducing this design with very limited/absence of media queries. Some of the issues above will be solved with media queries.

Please bring the changes above and re-submit review form

**Comment by mosesmuchemi** [comment]
refactored to put in the changes recommended by TSE

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi Muchemi!  
#### Your project looks :+1:  and very discriptive readme file!
#### But need some Required Changes

## [OPTIONAL]
##### Footer

* [x] It contains Newsweek logo.
* [ ] It contains social media icons.
* [x] It contains main navigation links.
* [x] **It changes with RWD breakpoints.**

In "TOP STORY" section little error let`s fix it
*** you use col-5 ***
before-sm

*** if you use col-sm enstead of col-5 ***
after-sm


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1322 @@
...
+      <article class="article-post">
+        <div class="article-img"><a href="#"><img src="https://d.newsweek.com/en/full/1543201/moscowmitchisnext-trends-kentucky-election-bevin.jpg?w=397&h=265&q=90&f=ba83e1ffa7dafb123d2312e3d112c341" alt="mc-commel" class="img-fluid"> </a>
+        <span class="sub-title">US</span></div>
+        <h4 class="stories-title pt-2"><a href="#">Critics Taunt McConnell as Democrat Claims Victory in Kentucky Governor's Race</a></h4>
+        <p class="stories-desc">Mitch McConnell's critics saw Tuesday's election as a sign of the Senate majority leader's fate in 2020, but President Donald Trump argued it was proof he would "win big".</p>
+    </article>
+
+            </div>
+            
+            <div class="col-5 col-md-7 col-lg-5 top-story-col">
```
Comment (Anna-Myzukina) [inline index.html]: Better to use col-sm instead of col-5 ( line 129)

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Not displayed social media icons in the footer

**Comment by mosesmuchemi** [comment]
done corrections as suggested by second TSE reviewer

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1323 @@
...
+          <div class="col d-none d-md-block mx-3 pb-4"><span class="current-date">Mon, November 04, 2019</span></div>
+          <div class="col-4 col-md-4 img-logo"><a href="#"><img src="images/newsweeklogo.png" class="img-fluid"
+                alt="newsweek logo"></a></div>
+          <div class="col btn-col">
+            <button type="button" class="btn trans-btn d-block d-xl-none float-right pb-3"><i class="fa fa-bars" aria-hidden="true"></i></button>
+            <button type="button" class="btn float-right subscribe-btn text-danger bg-white mx-3 mb-4 pb-1">SUBSCRIBE <i class="fas fa-chevron-right"></i></button>
+            <button type="button" class="btn float-right mr-2 signin-btn text-white pb-4">SIGN IN</button>
+      
+          </div>
+        </div>
```
Comment (suretrust) [inline index.html]: The width of the navbar on all screens is inaccurate. Please see the images below:

**Yours**
navbar
navbar2
navbar3

**Original**
nav-def
nav-def-2
nav-def-3

Comment (suretrust) [reply index.html]: The navigation links/buttons alignments are also inaccurate on a medium screen.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @mosesmuchemi,

### Great job thus far, :raised_hands: just one more suggestion to wrap it up. Please see my comments on your code.

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Comment by nikasulo** [comment]
# Fantastic job @mosesmuchemi :rocket: 
Remember to keep this energy going throughout the program, you'll be awesome for it :smile: 

#### Please merge this beautifully done job and move on with your projects!

**Review by nikasulo — APPROVED** [review]

**Pull request merged**